.github/workflows/cherryPick.yml

```diff
@@ -79,31 +64,38 @@ jobs:
           USER: OSBotify
           TITLE_REGEX: Update version to ${{ env.NEW_VERSION }}
 
-      - name: Cherry-pick the version-bump to new branch
+      - name: Create branch for new pull request
         run: |
-          git fetch
-          git cherry-pick -S -x --mainline 1 --strategy=recursive -Xtheirs ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }}
+          SOURCE_MERGE_BASE=$(git merge-base staging ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }})
+          VERSION_BUMP_MERGE_BASE=$(git merge-base staging ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }})
+          CP_MERGE_BASE=$(git merge-base "$SOURCE_MERGE_BASE" "$VERSION_BUMP_MERGE_BASE")
+          git checkout -b ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }} "$CP_MERGE_BASE"
+          git push --set-upstream origin ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }}
 
-      - name: Cherry-pick the merge commit of target PR to new branch
-        id: cherryPick
-        run: |
-          echo "Attempting to cherry-pick ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}"
-          if git cherry-pick -S -x --mainline 1 ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}; then
-            echo "🎉 No conflicts! CP was a success, PR can be automerged 🎉"
-            echo "::set-output name=SHOULD_AUTOMERGE::true"
-          else
-            echo "😞 PR can't be automerged, there are merge conflicts in the following files:"
-            git --no-pager diff --name-only --diff-filter=U
```

Comment:
This was a pretty weird workaround – we were literally committing the conflicts including the `====HEAD====` stuff, and asking people to overwrite that.

Fortunately, I don't think we'll really need to do this anymore. Instead, we'll CP the commits into a temporary branch which won't have any conflicts, then create the PR. If the PR has merge conflicts, then it will be unmergeable and we'll assign someone to resolve those conflicts. However, it will appear like a normal PR with merge conflicts listed at the bottom of the page. None of the wonky `====HEAD====` stuff will be committed into the files.